archetypes/Portfolio/Queued/index.tsx

```diff
@@ -168,8 +170,7 @@ const CommitRow: React.FC<
                     <div>
                         <div className="flex">
                             <div>
-                                {token.symbol.split('-')[0][0]}-
-                                {token.symbol.split('-')[1].split('/')[0] === 'BTC' ? 'Bitcoin' : 'Ethereum'}
+                                {token.symbol.split('-')[0][0]}-{base === 'BTC' ? 'Bitcoin' : 'Ethereum'}
```

Comment:
looks like it was already an issue but I don't think this supports assets other than Bitcoin and Etehreum. Might want something like https://github.com/tracer-protocol/pools-client/blob/develop/archetypes/Portfolio/History/index.tsx#L174